Capstone/Assets/Game/Scripts/Enemy.cs

```diff
@@ -60,8 +66,15 @@ public virtual void Awake()
 
         public virtual void Start()
         {
+            AIDirector.Instance.AddToEnemyGroup(this);
+
             player = GameObject.Find("Player").GetComponent<Player>();
-            
+
+            moveVector = Vector3.zero;
+            moveSpeed = 5f;
```

Comment:
Do we want to set this here? Or is this something we'll want to just set in the inspector?